uri-test

```diff
@@ -3,6 +3,8 @@
 use strict;
 use warnings;
 
+use lib ( './lib' ); #-- use local implementation while authoring module
+
```

Comment:
Did you want to remove that again? I've not noticed this file before, but it seems like a useful change to me.

Reply:
This looks like a simple interactive tool to check various URIs and methods. Quite useful to experiment with.
However, it seems that my environment was setup in such a way that my own perl installation was used and
thus I checked the installed library - not the one under development.
I would keep the change. Does no harm. 
Perhaps a relic of the security change that happened in 5.26 (?) where the '.' was removed from `@INC`.?

Reply:
It's often useful to run the against other versions of the module, such as the version already installed. This prevents that from working. And it can easily be run with `perl -Ilib`. But I'm not particularly concerned about this script either way.

Reply:
Just to avoid potentially breaking other peoples expectations on how this program should behave, I'll remove this and just issue a warning in case ./lib is not used and the program was invoked interactively. 
That visual hint would have saved me 10 minutes to find out why my changes had no effect.